src/features/import-projects/helpers.ts

```diff
@@ -47,63 +47,96 @@ export const updateOrCreateProject = async (
 
   // Remove project if prelimResult is "Remove"
   if (prelimResult && project[prelimResult] === "Remove") {
-    // If project is imported, delete it from the database
-    if (existingProject?.imported) {
-      try {
-        await dataSource
-          .createQueryBuilder()
-          .delete()
-          .from(Project)
-          .where("id = :id", { id })
-          .execute();
+    if (existingProject) {
+      // Remove the current round from rfRounds
+      const updatedRfRounds = (existingProject.rfRounds || []).filter(
+        (rfr) => rfr !== rfRound
+      );
 
-        console.log(
-          `[${new Date().toISOString()}] - INFO: Project Deleted. Project ID: ${id}`
-        );
-      } catch (error) {
-        // make project as not imported if failed to delete because it may have some vouches
-        console.log(
-          `[${new Date().toISOString()}] - ERROR: Failed to delete project. Project ID: ${id}`
-        );
-        const updatedProject = new Project({
-          ...existingProject,
-          imported: false,
-        });
+      if (updatedRfRounds.length === 0) {
+        // If no rounds remain, delete the project
+        if (existingProject.imported) {
```

Comment:
It seems possible to have projects with `imported = false`. But in the new code, it's tried to delete the projects, otherwise, make the `imported = false`. 
As I remember,  one way to have `imported=false` is indexing on chain attestation to a project that is not imported. 
If having an attestation is the only reason not to keep a project, let's do the logic explicitly and check whether it has any connected attestation, then decide about deletion.
BTW, I think it may work to only set `imported = false` as a safer approach